src/sqlfluff/rules/L031.py

```diff
@@ -71,12 +76,29 @@ class Rule_L031(BaseRule):
 
     """
 
+    config_keywords = ["force_enable"]
```

Comment:
```suggestion
    config_keywords = ["force_enable"]
    _dialects_disabled_by_default = ["bigquery"]
```

Reply:
Ah. You initially said "not a private class variable". Typo?

WIll change. 👍

Reply:
> Ah. You initially said "not a private class variable". Typo?

Was thinking about writing "in a hardcoded, private class variable, but no need to expose it as a fully configurable `.sqlfluff` configuration. What came out of the keyboard was a bit of that :-)